src/main/java/com/socrata/api/Soda2Producer.java

```diff
@@ -76,6 +78,7 @@ public static final Soda2Producer newProducerWithRequestId(final String url, Str
     public Soda2Producer(HttpLowLevel httpLowLevel)
     {
         super(httpLowLevel);
+        factory = new JsonFactory();
```

Comment:
OK, I guess the standard procedure is to edit this file with my credentials, but do not check those changes in

I tried this and while more tests do pass, some now report authorization problems

```
[info] Test com.socrata.api.SodaImporterTest.testCreatingFileDataset started
[error] Test com.socrata.api.SodaImporterTest.testCreatingFileDataset failed: You aren't allowed to create a dataset within this domain.
[error]     at com.socrata.api.HttpLowLevel.processErrors(HttpLowLevel.java:689)
[error]     at com.socrata.api.HttpLowLevel.postFileRaw(HttpLowLevel.java:532)
[error]     at com.socrata.api.SodaImporter$4.issueRequest(SodaImporter.java:570)
[error]     at com.socrata.api.SodaImporter.importNonDataFile(SodaImporter.java:590)
[error]     at com.socrata.api.SodaImporterTest.testCreatingFileDataset(SodaImporterTest.java:425)
```